libs/galley-types/src/Galley/Types/Teams.hs

```diff
@@ -258,11 +259,13 @@ instance ToJSON FeatureSSO where
 instance FromJSON FeatureLegalHold where
   parseJSON (String "disabled-permanently") = pure $ FeatureLegalHoldDisabledPermanently
   parseJSON (String "disabled-by-default") = pure $ FeatureLegalHoldDisabledByDefault
+  parseJSON (String "whiteliste-teams-and-implicit-consent") = pure FeatureLegalHoldWhitelistTeamsAndImplicitConsent
```

Comment:
```suggestion
  parseJSON (String "whitelisted-teams-and-implicit-consent") = pure FeatureLegalHoldWhitelistedTeamsAndImplicitConsent
```

Reply:
Thanks for the typo, but I prefer imperative over passive (am I getting the terminology right here?)

Reply:
0566e30500d6d1d3c5645253d4aa49649e61215f